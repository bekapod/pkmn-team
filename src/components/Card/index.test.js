// @flow
import React from "react";
import renderer from "react-test-renderer";
import { CardWrapper, CardHeader, CardContent } from ".";

describe("Card components", () => {
  it("renders without crashing", () => {
    const tree = renderer.create(
      <CardWrapper>
        <CardHeader types={["ELECTRIC", "PSYCHIC"]}>Heading</CardHeader>
        <CardContent>Content</CardContent>
      </CardWrapper>
    );

    expect(tree.toJSON()).toMatchSnapshot();
  });

  it("renders without crashing when types are not passed", () => {
    const tree = renderer.create(
      <CardWrapper>
        <CardHeader>Heading</CardHeader>
        <CardContent>Content</CardContent>
      </CardWrapper>
    );

    expect(tree.toJSON()).toMatchSnapshot();
  });
});
