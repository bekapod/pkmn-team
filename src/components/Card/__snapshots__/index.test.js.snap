// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card components renders without crashing 1`] = `
<article
  className="Card__CardWrapper-sc-1prdgex-0 hFLESf"
>
  <header
    className="Card__CardHeader-sc-1prdgex-1 gxtGbS"
  >
    Heading
  </header>
  <div
    className="Card__CardContent-sc-1prdgex-2 cyIVmL"
  >
    Content
  </div>
</article>
`;

exports[`Card components renders without crashing when types are not passed 1`] = `
<article
  className="Card__CardWrapper-sc-1prdgex-0 hFLESf"
>
  <header
    className="Card__CardHeader-sc-1prdgex-1 jVbTxc"
  >
    Heading
  </header>
  <div
    className="Card__CardContent-sc-1prdgex-2 cyIVmL"
  >
    Content
  </div>
</article>
`;
